src/Microsoft.AspNet.Mvc.Razor/Logging/RoslynCompilationServiceLoggerExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Mvc.Logging
+{
+    public static class RoslynCompilationServiceLoggerExtensions
+    {
+        private static readonly double TimestampToTicks = Stopwatch.Frequency / 10000000.0;
+        private static readonly Action<ILogger, string, Exception> _generatedCodeToAssemblyCompilationStart;
+        private static readonly Action<ILogger, string, double, Exception> _generatedCodeToAssemblyCompilationEnd;
+
+        static RoslynCompilationServiceLoggerExtensions()
+        {
+            _generatedCodeToAssemblyCompilationStart = LoggerMessage.Define<string>(
+                LogLevel.Debug,
+                1,
+                "Compilation of the generated code for the Razor file at '{FilePath}' started.");
```

Comment:
We don't have Razor specific logging elsewhere, so no. However we do use that term in our docs
For .e.g
> render a view that uses Razor syntax